### ISSUE TITLE
Colper Science: Crowdfunding & Content Extraction from OA Publications

### ISSUE THREAD

**Issue by KambizChizari** [mozilla/global-sprint#62]
**[ Project Contact ]** Kambiz Chizari
**[ GitHub Repo ]** https://github.com/ColperScience
**[ Track ]** Open Innovation
**[ Community ]** Mozilla Science - wrangler: @stephwright
**[ Location ]** Montreal - Canada (ET)
### Description
We are developing an Open Access (OA) interactive platform called Colper Science with two major goals:
1- to advance the scientific publishing system towards a more open and efficient system.
2- to facilitate the connection between the general public and scientific publications and the research community.
 We would like to share two sprints related to the project:
1- Crowdfunding: Spreading the word & help with the video
2- Discover different ways of content extraction from OA publications 
***

## Want to contribute to this project during #mozsprint?
Join us at the [Global Sprint, June 1-2](https://mozilla.github.io/global-sprint). Leave a comment below if you're interested in contributing to this project during #mozsprint 2017!

***

## Note to the Project Lead :tada:
Congrats, Kambiz Chizari! This is your official project listing for the [Mozilla Global Sprint 2017](https://mozilla.github.io/global-sprint). To confirm your registration, please complete and check off the following:

- [x] Complete [Open Leadership 101](https://mozilla.teachable.com/p/open-leadership-101)
- [x] Provide a GitHub repository for work and discussion on your project in a comment
  * new to GitHub?  Here's a [step-by-step guide on using the #mozsprint template](https://mozilla.github.io/global-sprint/project-lead/templates/)
- [x] Create a README file in your project repository. This file should help newcomers understand what your project is, why it's important, and kinds of help you're looking for.
* [x] [Create file: LICENSE](http://choosealicense.com/) to give your project an open license, allowing for sharing and remixing.
- [ ] Turn on your [Issue Tracker](https://mozilla.github.io/global-sprint/project-lead/templates/#4-create-issues) and create issues to describe each task that you need help with and how a contributor can get started on that task. [Create a label](https://mozilla.github.io/global-sprint/project-lead/templates/#5-add-a-mozsprint-label) called `mozsprint` and apply it to your issues.

#### Checklist for FEATURED Projects :clipboard:
To have your project FEATURED on [Mozilla Pulse](http://mozillapulse.org/), complete the following documentation. In past Sprints, well-documented featured projects have 5 times more contributions than other projects. Details about each item and more information about how to create them are [on our Project Requirements Page](https://mozilla.github.io/global-sprint/project-lead/project-requirements/). 

* [ ] In your README, link to the [Mozilla Community Participation Guidelines](https://www.mozilla.org/en-US/about/governance/policies/participation/) or write your own code of conduct.
* [ ] Create file: CONTRIBUTING.md so others know how they can contribute. If you'd like, you can [remix this template](https://github.com/acabunoc/mozsprint-repo-template/blob/master/CONTRIBUTING.md)
* [ ] Comment with an image to use on Mozilla Pulse for this project listing

Once all of the above is complete,
- [ ] Leave a comment with the text `This is ready for Mozilla Pulse`. Your community wrangler will review this issue and post your project :balloon:

If you get stuck at any point, feel free to look at the [requirements page](https://mozilla.github.io/global-sprint/project-lead/project-requirements/) and [project templates](https://mozilla.github.io/global-sprint/project-lead/templates/) or reach out to your community wrangler, @stephwright. We're here to help you through this process.

**Comment by stephwright** [comment]
Hi @KambizChizari ... Global Sprint is only 3 weeks away and I wanted to check in on your preparation  progress.  Looks like you just need to put some work into your CONTRIBUTING.md, add some issues, link to our Participation Guidelines and get me an image to be featured on Pulse: https://www.mozillapulse.org/latest

At a minimum I'd recommend working on your CONTRIBUTING and adding some issues so folks know how to help you on your project during the sprint.

**Comment by KambizChizari** [comment]
Hi @stephwright ... Since we are postponing the crowdfunding and also an we will us an API for the content extraction, we should redefine which parts of the project can be done in a collaborative way. We discussed it with our mentor, Bastian, and decided to discuss regarding the future plans with Abby @acabunoc in our next meeting (18 May). We are planning to build a community related to open access publications before our crowdfunding and for that we are going to interview people who are active in the field or who conduct research using open data and share it as videos on YouTube and as audio podcast. So after our discussion on 18 May things will be clearer and the GitHub will be updated then.

**Comment by stephwright** [comment]
Perfect!  Sounds like you have a plan in place for going forward.  Thanks for the update!

**Comment by ilyasst** [comment]
@stephwright Is it possible to deleted this issues? Our project changed, so I created a new issue describing it here: https://github.com/mozilla/global-sprint/issues/126

**Comment by stephwright** [comment]
Hi @ilyasst I closed the issue.
